commit 3bd77e51758cf9ebdeee1f4d140167bde6adcded (refs/changes/76/1876/4)
Author: Brian Bennett <brian.bennett@joyent.com>
Date:   2017-05-10T22:04:04+00:00 (2 years, 5 months ago)
    
    MANTA-3230 muppet should remove RC4, 3DES.
    Reviewed by: Alex Wilson <alex.wilson@joyent.com>
    Approved by: Alex Wilson <alex.wilson@joyent.com>

diff --git a/etc/stud.conf b/etc/stud.conf
index a2417ff..ed9a35e 100644
--- a/etc/stud.conf
+++ b/etc/stud.conf
@@ -34,9 +34,9 @@ pem-file = "/opt/smartdc/muppet/etc/ssl.pem"
 # Run openssl ciphers for list of available ciphers.
 # type: string
 #
-# Preferred cipher list taken from
-# https://github.com/cloudflare/sslconfig
-ciphers = "EECDH+CHACHA20:EECDH+AES128:RSA+AES128:EECDH+AES256:RSA+AES256:EECDH+3DES:RSA+3DES:HIGH:RC4-SHA:!MD5:!aNULL:!PSK"
+# Preferred cipher list as per RFD 93
+# https://github.com/joyent/rfd/tree/master/rfd/0093
+ciphers = "EECDH+CHACHA20:EECDH+AES128:RSA+AES128:EECDH+AES256:RSA+AES256:HIGH:!MD5:!aNULL:!PSK"
 
 # Enforce server cipher list order
 #
@@ -121,3 +121,5 @@ write-proxy = on
 #
 # type: boolean
 proxy-proxy = off
+
+# EOF
